models/event_filters/stripe_charges.sql

```diff
@@ -5,7 +5,7 @@ with charges as (
 )
 
 select
-  id,
+  data__object__id as id,
```

Comment:
This could work. `id` is the ID for the event; `data__object__id` is the ID for the charge. This will NOT be unique with your current WHERE statement (see next comment)